torq.q

```diff
@@ -6,6 +6,11 @@ loaded:1b
 // Initialised flag - used to check if the process is still initialisation
 initialised:0b
 
+// function to add functions to initialisation list
+initlist:()
+addinitlist:{[x]if[1<count x;.proc.initlist,:enlist x]};
+  
+//add to init list functions explicitly defined on command line
```

Comment:
Is this comment necessary? It looks like it should be the other side of the new code, if it is still valid.

Reply:
This comment was supposed to be deleted